### DISCUSSION TITLE
The "emoji reaction" button is too large and increases the height of the comment item for no profit

### DISCUSSION THREAD

**Discussion by Nakilon** [community/community#3278] [Discussions]
![image](https://user-images.githubusercontent.com/2870363/106385862-5cc3e480-63e3-11eb-994d-2ebc076e20a3.png)

Also move it to the right, please, because the left side of any comment entity in web forms of forums are more for attributes like nickname, userpic, timestamp", etc.

**Comment by PranamBhat** [comment]
Hello!

There are scenario's like for example:

1) User will come to Discussion section and go for particular discussion and just react for a comment and comes back.

2) User will read the comment and respond to it with a reply window.

3) User will react and respond to the comment with a reply window.


Your scenario will occur at 3rd stage. Here, user will react and reply as well. It is not going to be a problem. As soon as user reacts using reaction window, it will close and user can write a reply over response window. Moreover, this response window occupying the space of reaction window if you notice. It will be good sign when it comes to **UI Experience** and **Web Page Space Consumption**.

Attaching the screenshot for reference.


![image](https://user-images.githubusercontent.com/32120033/108023363-6175d300-7048-11eb-86f2-56eca5d3816b.png)

**Reply by Nakilon** [reply]
In my opinion the button should not be there in either scenario.
Github isn't Facebook where giving the Likes is the main purpose of the website to aggregate and then sell the personal psychological portrait data. People are in Github for discussing, not reacting. The button should be somewhere else and not adding to the vertical size of the interface.

**Reply by Levi-Lesches** [reply]
Fully agree with @Nakilon here. Especially with the addition of traditional upvotes, the different reactions are often a bit "too much", and I frequently see odd uses: 
- 😕 ("confused") to mean "I don't like it"
- 🚀 ("ship this feature") being used to mean "congrats"
- 😄 ("smile") to mean "wow this repo maintainer doesn't follow up"

**Reply by Nakilon** [reply]
This is pain...
![image](https://user-images.githubusercontent.com/2870363/180978830-4c6706ab-9506-4ae6-b231-67715d7cfc0a.png)
